client-assertion-validation/src/test/scala/it/pagopa/interop/clientassertionvalidation/ClientAssertionValidationSpec.scala

```diff
@@ -91,6 +91,14 @@ class ClientAssertionValidationSpec extends AnyWordSpecLike {
       ) shouldBe Left(NonEmptyList.one(InvalidPurposeIdFormat("not-an-uuid")))
     }
 
+    "fail when purpose id is ampty string" in {
+      val assertion = fastClientAssertionJWT(customClaims = Map(PURPOSE_ID_CLAIM -> ""))
```

Comment:
To test what I suggested
```suggestion
      val assertion = fastClientAssertionJWT(customClaims = Map(PURPOSE_ID_CLAIM -> " 
  "))
```